google/cloud/bigquery/schema.py

```diff
@@ -796,8 +792,6 @@ def serde_info(self) -> Any:
         prop = _get_sub_prop(self._properties, ["serDeInfo"])
         if prop is not None:
             prop = StorageDescriptor().from_api_repr(prop)
-            print(f"DINOSAUR prop: {prop}")
```

Comment:
Lol, oops! 😆 Could be worth a separate cleanup PR to have something small and easy to approve.

Reply:
Removed.